### PR TITLE
styled-system: Add individual border and overflow props

### PR DESCRIPTION
**Pull request by eliseumds**
New props introduced in styled-system@5.1.0

Docs of border props: https://styled-system.com/api/#border
Docs of overflow props: https://styled-system.com/api#layout

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes

### CODE REVIEW

**Review by peduarte — APPROVED** [review]

**Comment by peduarte** [comment]
LGTM 👌

File: types/styled-system/index.d.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@
 //                 Dhalton Huber <https://github.com/Dhalton>
 //                 Elliot Bonneville <https://github.com/elliotbonneville>
 //                 Jack Caldwell <https://github.com/jackcaldwell>
+//                 Eliseu Monar dos Santos <https://github.com/eliseumds>
```
Comment (PranavSenthilnathan) [inline types/styled-system/index.d.ts]: Rev up the version to 5.1 at the top of the file.

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

**Comment by eliseumds** [comment]
@PranavSenthilnathan done. Tks.

**Review by PranavSenthilnathan — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**